cmd/main_test.go

```diff
@@ -221,6 +392,58 @@ func prepareMysql() {
 	}
 }
 
+func prepareOracle() {
+	db, err := sql.Open("godror", "oracle://a:123@localhost:49161/XE")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer db.Close()
+	_, err = db.Exec("BEGIN EXECUTE IMMEDIATE 'DROP USER mydb CASCADE'; EXCEPTION WHEN OTHERS THEN IF SQLCODE = -1918 THEN NULL; ELSE RAISE; END IF; END;")
+	if err != nil {
+		log.Fatal(err)
+	}
+	_, err = db.Exec(fmt.Sprintf("CREATE USER mydb IDENTIFIED BY mydb DEFAULT TABLESPACE USERS"))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	_, err = db.Exec(fmt.Sprintf("GRANT DBA TO mydb"))
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Create table
+	_, err = db.Exec(`
+		CREATE TABLE mydb.test_table (
+			id NUMBER(10) PRIMARY KEY,
+			int_col NUMBER(10),
+			varchar_col VARCHAR2(255),
+			float_col NUMBER,
+			bool_col NUMBER(1) CHECK (bool_col IN (0, 1)),
+			de NUMBER(18,6),
+			date_col DATE,
+			datetime_col TIMESTAMP,
+			timestamp_col TIMESTAMP
+		)
+	`)
+	// need to test the TIME type in mysql
```

Comment:
`// need to test the TIME type in mysql` this comment can be removed here :)

Reply:
Done